packages/node_modules/@webex/plugin-meetings/src/meetings/index.js

```diff
@@ -605,12 +622,27 @@ export default class Meetings extends WebexPlugin {
       });
     }
 
+    /**
+     * Fetch user prefered webex site information
+     * This also has other infomation about the user
+     * @returns {Promise}
+     * @private
+     * @memberof Meetings
+     */
+    fetchUserPreferredWebexSite() {
+      return this.request.fetchLoginUserInfomation().then((res) => {
+        this.parseUserPreferences = MeetingsUtil.parseparseUserPreferences(res.userPreferences);
```

Comment:
why is it called `parseUserPreferences` if it contains only the webex site? it should be called `webexSite`, right? or if it's supposed to be parsed preferences then it should be an object with webexSite being one of the fields in it. Also it's not initialised in constructor and missing the comments like all other fields initialised in constructor have.

Reply:
`{"schemas":["urn:scim:schemas:core:1.0","urn:scim:schemas:extension:cisco:commonidentity:1.0"],"userName":"arungane@cisco.com","emails":[{"primary":true,"type":"work","value":"arungane@cisco.com"}],"name":{"givenName":"Arun","familyName":"Ganeshan"},"phoneNumbers":[{"type":"work","value":"+1 408-902-3322"},{"type":"mobile","value":"+1 425-208-6070"}],"addresses":[{"type":"work","streetAddress":"","locality":"RESEARCH TRIANGLE PARK","region":"NORTH CAROLINA","postalCode":"","country":"US"}],"entitlements":["insights-service","ucm-calling","atlas-portal","basic-meeting","basic-message","spark-scrums-member","squared-call-initiation","squared-fusion-cal","squared-fusion-mgmt","squared-room-moderation","squared-room-moderation-wbxlinked","squared-syncup","squared-syncup-wbxlinked","squared-team-member","webex-ecm","webex-squared","webex-voice-assistant","cloudmeetings","screen-share","spark","spark--media","spark-admin","journal-tool","spark-beta-access","spark-dashboard","spark-devconsole"],"title":"Software Engineering Technical Leader","cisSyncSource":"DirSync","photos":[{"type":"photo","value":"https://avatar-prod-us-east-2.webexcontent.com/Avtr~V1~1eb65fdf-9643-417f-9974-ad72cae0e10f/V1~d97af9524de60a48648f5256af571fe1e2e624add77ee596ed328e7d2055e4d2~e7414c3ebe1748149839fd3f0cb2eb4b~1600"},{"type":"thumbnail","value":"https://avatar-prod-us-east-2.webexcontent.com/Avtr~V1~1eb65fdf-9643-417f-9974-ad72cae0e10f/V1~d97af9524de60a48648f5256af571fe1e2e624add77ee596ed328e7d2055e4d2~e7414c3ebe1748149839fd3f0cb2eb4b~80"}],"id":"344ea183-9d5d-4e77-aed2-c4f004afd915","meta":{"created":"2016-04-07T15:48:35.354Z","lastModified":"2022-03-31T03:00:01.803Z","lastLoginTime":"2022-04-08T14:07:50.352Z","version":"W/\"19078568565\"","location":"https://identity.webex.com/identity/scim/1eb65fdf-9643-417f-9974-ad72cae0e10f/v1/Users/344ea183-9d5d-4e77-aed2-c4f004afd915","organizationID":"1eb65fdf-9643-417f-9974-ad72cae0e10f"},"department":"020029745","ims":[{"type":"webex-messenger-jid","value":"arungane@cisco.com"},{"type":"webex-messenger-jid","value":"arungane@cisco.com","primary":true},{"type":"webex-squared-jid","value":"arungane@cisco.com"},{"type":"xmpp","value":"arungane@cisco.com"},{"type":"xmpp-fed-jid","value":"arungane@cisco.com"}],"displayName":"Arun Ganeshan","roles":["id_readonly_admin","journal-tool.journal_dev","atlas-portal.wbxsiteadmin","atlas-portal.partner.helpdesk","atlas-portal.partner.helpdesk.advanced"],"active":true,"licenseID":["CF_0b862d36-3535-4f76-8aa1-6d36b6791a66","MS_c0887c4a-c131-439d-86b6-bc2c7ab9a94c","WAM_8261f691-9e81-47bc-9ea3-1631132e2710","EE_82fd77f1-9f14-440a-82a1-41b1c4d20c0b_co.webex.com"],"userSettings":["{\"callingBehavior\":\"NATIVE_SIP_CALL_TO_UCM\"}","{\"user-origin\":\"admin-invited\"}","{\"spark.signUpDate\":\"1460044115354\"}"],"userPreferences":["SparkTOSAccept","\"preferredWebExSite\":\"go.webex.com\""],"sipAddresses":[{"type":"cloud-calling","value":"arungane@cisco.call.ciscospark.com","primary":true},{"type":"enterprise","value":"arungane@cisco.com","primary":true},{"type":"personal-room","value":"arungane@businessroundtable.webex.com"},{"type":"personal-room","value":"20015353601@co.webex.com"},{"type":"personal-room","value":"arungane@co.webex.com"},{"type":"personal-room","value":"122000484@alpha-fed.webex.com"},{"type":"personal-room","value":"1269139850@cisco-collab.webex.com"},{"type":"personal-room","value":"128497494@alpha.webex.com"},{"type":"personal-room","value":"150902919@go.webex.com"},{"type":"personal-room","value":"203725286@acecloud.webex.com"},{"type":"personal-room","value":"205719228@cisco.webex.com"},{"type":"personal-room","value":"arungane@acecloud.webex.com"},{"type":"personal-room","value":"arungane@alpha-fed.webex.com"},{"type":"personal-room","value":"arungane@alpha.webex.com"},{"type":"personal-room","value":"arungane@cisco-collab.webex.com"},{"type":"personal-room","value":"arungane@cisco.webex.com"},{"type":"personal-room","value":"arungane@go.webex.com"}],"managedOrgs":[{"orgId":"67f7eb2c-6c6e-47d4-a5df-a34c4c9c3928","role":"id_full_admin"}],"trainSiteNames":["co.webex.com","businessroundtable.webex.com"],"adminTrainSiteNames":["businessroundtable.webex.com#read-only"],"linkedTrainSiteNames":["acecloud.webex.com","alpha-fed.webex.com","alpha.webex.com","cisco-collab.webex.com","cisco.webex.com","go.webex.com"],"extLinkedAccts":[{"providerID":"acecloud.webex.com","accountGUID":"489214027","status":"active"},{"providerID":"alpha-fed.webex.com","accountGUID":"859262","status":"active"},{"providerID":"alpha.webex.com","accountGUID":"143782","status":"active"},{"providerID":"cisco-collab.webex.com","accountGUID":"595720262","status":"active"},{"providerID":"cisco.webex.com","accountGUID":"488741782","status":"active"},{"providerID":"go.webex.com","accountGUID":"499713752","status":"active"}],"manager":{"managerId":"4514d324-3a9d-4d32-bcd8-f0f8276eeba7","displayName":"David Hoff"},"isTeamsOnJabberEnabled":false,"isUCCallOnJabberEnabled":false,"templates":[{"type":"callingbehaviortemplate","value":"2d6de105-7a35-49e0-829b-59441a023f4c"}],"userType":"user","avatarSyncEnabled":false,"teamsClusterId":"urn:TEAM:us-east-2_a","urn:scim:schemas:extension:enterprise:1.0":{"department":"020029745","manager":{"managerId":"4514d324-3a9d-4d32-bcd8-f0f8276eeba7","displayName":"David Hoff"}}}`